modules/local/align_reads.nf

```diff
@@ -29,7 +29,7 @@ process ALIGN_READS {
         path "*.version.txt"                   , emit: version
 
     when:
-        !meta.cov_from_spades
```

Comment:
I think these `when` control flows should be handled within the calling workflow/subworkflow because I don't think it's unreasonable to think these modules might be reused for other subworkflows in the future. It also will probably allow defining that flow in a single place

Reply:
I agree. Although I'm not sure the `when` directive can be used within a workflow